ports/geckolib/string_cache/lib.rs

```diff
@@ -30,90 +33,88 @@ pub mod atom_macro;
 
 #[macro_export]
 macro_rules! ns {
-    () => { $crate::Namespace(atom!("")) };
+    () => { atom!("") }
 }
 
-#[derive(PartialEq, Eq)]
-pub struct Atom(*mut nsIAtom);
-#[derive(PartialEq, Eq, Debug, Hash, Clone)]
-pub struct Namespace(pub Atom);
+pub type Namespace = Atom;
```

Comment:
This was mostly to get things building more easily. I can re-introduce the newtype and add `WeakNamespace`.